dynatrace/dynatrace.go

```diff
@@ -52,6 +54,18 @@ func NewExporter(opts Options) (*Exporter, error) {
 
 	client := &http.Client{}
 
+	if opts.Tags == nil {
+		opts.Tags = []mint.Tag{}
+	}
+
+	// todo use a key-value pair here.
+	if opts.EnrichWithOneAgentMetadata {
```

Comment:
Should this be true or false by default? /cc @arminru

Reply:
For the SDK exporters I'd go with true, which is already the case in Java and Python.
It's only evaluated once at startup and in Go on Linux it will fail just as quickly as if there was no OneAgent present in the other exporters.